src/main/java/org/ibp/api/java/impl/middleware/dataset/validator/ObservationValidator.java

```diff
@@ -54,16 +54,16 @@ public void validateObservation(final Integer datasetId, final Integer observati
 		}
 
 		if (observationDto != null) {
-			this.validateObservationValue(phenotype.getObservableId(), observationDto.getValue());
+			this.validateVariableValue(phenotype.getObservableId(), observationDto.getValue());
 			if (observationDto.getDraftValue() != null) {
-				this.validateObservationValue(phenotype.getObservableId(), observationDto.getDraftValue());
+				this.validateVariableValue(phenotype.getObservableId(), observationDto.getDraftValue());
 			}
 		}
 	}
 
-	public void validateObservationValue(final Integer variableId, final String value) {
+	public void validateVariableValue(final Integer variableId, final String value) {
```

Comment:
Why are there changes to `ObservationValidator` class which is used by `DatasetService`? Seems not relevant